### ISSUE TITLE
Infinite loading example has some bugs.

### ISSUE THREAD

**Issue by matamatanot** [vercel/swr#579]
# Bug report
https://swr.vercel.app/examples/infinite-loading

## Description / Observed Behavior

- When you press a `refresh` button(mutate), nothing has changed in the UI.
- When you press a `load more` button(setSize), duplicate issues are displayed.

## Expected Behavior

- When you press a `refresh` button(mutate), all the issues are gone.
- When you press a `load more` button(setSize), duplicate issues are **not** displayed.

## Repro Steps / Code Example

https://swr.vercel.app/examples/infinite-loading

## Additional Context

SWR version: 0.3.0

**Comment by shuding** [comment]
Thanks for reporting! I just fixed the duplicates. It was because GitHub's pagination API starts from 1 for the `?page=` param (https://developer.github.com/v3/guides/traversing-with-pagination/#navigating-through-the-pages). So `?page=0` and `?page=1` returned the same data.

For the `refresh` button, it's for scenarios like Twitter/Facebook's timeline refresh (check if there's any new item), so it doesn't clear the list. To avoid the confusion I added a `clear` button, hope that helps!
